docs/7.x/requirements.md

```diff
@@ -257,7 +253,6 @@ Gravity requires that these modules are loaded prior to installation.
 | Linux Distribution                     | Version | Modules |
 |--------------------------|-----------|---------------------------|
 | CentOS                    | 7.2     | bridge, ebtable_filter, iptables, overlay  |
-| RedHat Linux | 7.2     | bridge, ebtable_filter, iptables  |
 | CentOS                  | 7.3-7.6     | br_netfilter, ebtable_filter, iptables, overlay  |
 | RedHat Linux | 7.3-7.6     | br_netfilter, ebtable_filter, iptables, overlay     |
```

Comment:
Should this be bumped to 7.4?  I don't understand why it is 7.3 currently, while most of the rest of our documenatation seemed to split `7.2-7.3` vs `7.4+`